sdk/eventhub/eventhubs-checkpointstore-blob/samples/v1/javascript/sample.env

```diff
@@ -0,0 +1,18 @@
+# Used in most samples
+EVENTHUB_CONNECTION_STRING=<connection string WITHOUT an EntityPath>
+EVENTHUB_NAME=<name of a single eventhub>
+EVENTHUB_FQDN=<your-eventhubs-namespace>.servicebus.windows.net
+CONSUMER_GROUP_NAME=<name of a consumer group>
+STORAGE_CONNECTION_STRING=<connection string for a storage account for the checkpoint store>
+
+# Used in the useWithIotHub.ts sample
+IOTHUB_EH_COMPATIBLE_CONNECTION_STRING=<connection string with EntityPath>
```

Comment:
I don't _think_ this is needed for eventhubs-checkpointstore-blob.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/15137/commits/1fe41b3e6b340071861a97d13d4f948266dea2e7